### PR TITLE
fix: Renaming Landing Page

### PR DESCRIPTION
**Pull request by CryptoKnightIOG**
https://input-output.atlassian.net/browse/ATL-7062

### CODE REVIEW

**Review by yshyn-iohk — APPROVED** [review]
well done!

File: documentation/docs/quick-start.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ A Cloud Agent can issue, hold, and verify [verifiable credentials (VCs)](/docs/c
 
 It is maintained as an open source lab through the [Hyperledger Foundation: Open Enterprise Agent](https://labs.hyperledger.org/labs/open-enterprise-agent.html).
 
-More in depth documentation about [Cloud Agent](/docs/concepts/glossary/#cloud-agent-aka-prism-cloud-agent) can be found [here](/docs/atala-prism/prism-cloud-agent/overview).
+More in depth documentation about [Cloud Agent](/docs/concepts/glossary/#cloud-agent) can be found [here](/docs/identus/cloud-agent/overview).
```
Comment (mkbreuningIOHK) [inline documentation/docs/quick-start.md:40]: The link /docs/identus/cloud-agent/overview does not exist. You may do in 2 steps:
- rename the text only 
- update the links

File: documentation/docs/quick-start.md [hunk 2]
```diff
@@ -51,28 +51,28 @@ More in-depth documentation about the different Wallet SDKs can be found here ([
...
 
 
 
 ## PRE-REQUISITES
 
 
 ### Agent Deployment
-This guide will demonstrate a single-tenant deployment with API Key authentication disabled and an in-memory ledger for published DID storage, which is the simplest configuration to get started as a developer. More advanced configuration options can be found in [Multi-Tenancy Management](/tutorials/multitenancy/tenant-onboarding) and associated [Environment Variables](/docs/atala-prism/prism-cloud-agent/environment-variables) configuration options.
+This guide will demonstrate a single-tenant deployment with API Key authentication disabled and an in-memory ledger for published DID storage, which is the simplest configuration to get started as a developer. More advanced configuration options can be found in [Multi-Tenancy Management](/tutorials/multitenancy/tenant-onboarding) and associated [Environment Variables](/docs/identus/cloud-agent/environment-variables) configuration options.
```
Comment (mkbreuningIOHK) [inline documentation/docs/quick-start.md:65]: /docs/identus/cloud-agent/ does not exist yet

File: documentation/docs/quick-start.md [hunk 3]
```diff
@@ -256,15 +256,15 @@ curl -X 'POST' \
...
 
 ### Starting Sample App
-All wallet SDK's come bundled with a sample application, that cover all the PRISM flows, including establishing connections, issuance, and verification flows.
+All wallet SDK's come bundled with a sample application, that cover all the Identus flows, including establishing connections, issuance, and verification flows.
 
 <Tabs>
 <TabItem value="js" label="Typescript Sample APP">
 
-1. Clone the [TypeScript SDK](https://github.com/input-output-hk/atala-prism-wallet-sdk-ts) repository.
+1. Clone the [TypeScript SDK](https://github.com/input-output-hk/identus-edge-agent-sdk-ts) repository.
```
Comment (mkbreuningIOHK) [inline documentation/docs/quick-start.md:264]: Quick Start Guide is renamed by @todorkoleviohk.
Anyway, do not rename the wallet sdk repo as it is not existing yet. This is part of the actions to do after the migration to Hyperledger is done.

File: documentation/docs/quick-start.md [hunk 4]
```diff
@@ -256,15 +256,15 @@ curl -X 'POST' \
...
 
 ```bash
-git clone https://github.com/input-output-hk/atala-prism-wallet-sdk-ts
+git clone https://github.com/input-output-hk/identus-edge-agent-sdk-ts
```
Comment (mkbreuningIOHK) [inline documentation/docs/quick-start.md:267]: Do not rename the wallet sdk repo as it is not existing yet. This is part of the actions to do after the migration to Hyperledger is done.

File: docusaurus.config.js [hunk 5]
```diff
@@ -163,7 +163,7 @@ const config = {
                 ],
             },
             footer: {
-                copyright: `© ${new Date().getFullYear()} Atala PRISM. All rights reserved.`,
+                copyright: `© ${new Date().getFullYear()} Identus. All rights reserved.`,
```
Comment (mkbreuningIOHK) [inline docusaurus.config.js:166]: @essbante-io , how about the copyright to indicate or not?

File: docusaurus.config.js [hunk 6]
```diff
@@ -95,7 +95,7 @@ const config = {
         {
             navbar: {
                 logo: {
-                    alt: 'Atala PRISM logo',
+                    alt: ' Identus logo',
```
Comment (mkbreuningIOHK) [inline docusaurus.config.js:98]: @essbante-io , @petevielhaber , how about the logo?

**Review by mkbreuningIOHK — CHANGES_REQUESTED** [review]
@CryptoKnightIOG , 
- The Quick Start Guide is renamed by @todorkoleviohk , so please revert your changes.
- Some links are broken so the documentation is not building. You could do that in two steps: 1) renaming files only 2) changing directories. Up to you.

File: docusaurus.config.js (hunk 6 above)
Comment (petevielhaber) [reply docusaurus.config.js:98]: I will share PNG, SVG

File: documentation/docs/quick-start.md (hunk 2 above)
Comment (CryptoKnightIOG) [reply documentation/docs/quick-start.md]: it does now

File: documentation/docs/quick-start.md (hunk 1 above)
Comment (CryptoKnightIOG) [reply documentation/docs/quick-start.md]: fixed

File: documentation/docs/quick-start.md (hunk 3 above)
Comment (CryptoKnightIOG) [reply documentation/docs/quick-start.md]: fixed

File: documentation/docs/quick-start.md (hunk 4 above)
Comment (CryptoKnightIOG) [reply documentation/docs/quick-start.md]: fixed

**Review by mkbreuningIOHK — APPROVED** [review]

**Comment by mkbreuningIOHK** [comment]
We need to check the copyright and the logo in a later review, but let's merge this first iteration.

**Pull request merged**